supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -108,6 +108,39 @@
 
 *See also*: xref:ttl[TTL], xref:time-to-live-n[time to live]
 
+[discrete]
+[[tls]]
+==== image:images/yes.png[yes] TLS (noun)
+*Description*: TLS is an initialism for Transport Layer Security (TLS), and it is the successor to the Secure Sockets Layer (SSL) protocol.
+
+TLS is a cryptographic protocol that uses the Public Key Infrastructure (PKI) method to encrypt network traffic between two systems. PKI uses asymmetric encryption during a TLS handshake process to authenticate the connection between two systems.
+
+Use TLS when referring to protocols that exchange cryptographic keys and secure network connections between two systems. Check for the latest version of the TLS protocol and, if necessary, contact a subject matter expert (SME) to verify the TLS version to note in product documentation.
```

Comment:
```suggestion
Use "TLS" when referring to protocols that exchange cryptographic keys and secure network connections between two systems. Check for the latest version of the TLS protocol and, if necessary, contact a subject matter expert (SME) to verify the TLS version to note in product documentation.
```
Other glossary entries use quotation marks to refer to specific usage of terms.

Reply:
:heavy_check_mark: